picky-asn1-x509/CHANGELOG.md

```diff
@@ -7,7 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## [0.10.0] 2023-05-23
+### Added
```

Comment:
@CBenoit I was not sure -- should be bump crate versions again? We didn't release new picky crates from the previous PR, so maybe we should keep the version? (I'll update the changelog accordingly)

Reply:
Updated changelog